### PR TITLE
Em 279

### PR DESCRIPTION
**Pull request by eyalvo-matrix**

### CODE REVIEW

File: modules/EmergencyMedicine/sql/install.sql [hunk 1]
```diff
@@ -252,10 +252,16 @@ INSERT INTO `fhir_value_set_systems` (`vs_id`, `system`, `type`,`filter`)
...
 
 INSERT INTO `fhir_value_set_systems` (`vs_id`, `system`, `type`,`filter`)
-VALUES ('x_ray_types', 'x_ray_types', 'All', NULL);
+VALUES ('details_x_ray', 'details_x_ray', 'All', NULL);
+
+INSERT INTO `fhir_value_sets` (`id`, `title`)
+VALUES ('details_providing_medicine', 'Drug name');
+
+INSERT INTO `fhir_value_set_systems` (`vs_id`, `system`, `type`,`filter`)
+VALUES ('details_providing_medicine', '9911', 'Codes', NULL);
```
Comment (eyalvo-matrix) [inline modules/EmergencyMedicine/sql/install.sql:264]: notice 
const MOH_DRUGS = '9911'; for drugs
Controller/ImportDataController.php line 51

File: modules/EmergencyMedicine/sql/0_1_0-to-0_2_0_upgrade.sql [hunk 2]
```diff
@@ -322,15 +322,55 @@ INSERT INTO `fhir_value_set_systems` (`vs_id`, `system`, `type`,`filter`)
...
+VALUES ('details_x_ray', 'details_x_ray', 'All', NULL);
+#EndIf
+
+#IfRow2D list_options list_id lists option_id x_ray_types
 
+DELETE FROM list_options WHERE list_id="x_ray_types" OR option_id="x_ray_types"
+
+#EndIf
+
+#IfNotRow2D list_options list_id lists option_id details_x_ray
```
Comment (matrix-amiel) [inline modules/EmergencyMedicine/sql/0_1_0-to-0_2_0_upgrade.sql]: no reason to change in the list option table

I ask only to change the id and title of the value set

File: modules/EmergencyMedicine/sql/install.sql (hunk 1 above)
Comment (eyalvo-matrix) [reply modules/EmergencyMedicine/sql/install.sql:264]: "no reason to change in the list option table" - there is I told we work with const - good morning

File: modules/EmergencyMedicine/sql/0_1_0-to-0_2_0_upgrade.sql (hunk 2 above)
Comment (eyalvo-matrix) [reply modules/EmergencyMedicine/sql/0_1_0-to-0_2_0_upgrade.sql]: "no reason to change in the list option table" - there is as I told you in the phone ... we work with const...

if you want value set to be different then list option we need clinikal api that return the connection
so one can easily get it for qa and other purposes
and of course the filter-search option

**Pull request merged**